src/qt/trafficgraphwidget.cpp

```diff
@@ -9,7 +9,7 @@
 #include <QPainter>
 #include <QColor>
 #include <QTimer>
-
+#include <QPainterPath>
```

Comment:
Is this `include` necessary?

Reply:
It is not necessary now with QT5.9, but on my laptop with QT5.15.3 the build fails. It can be done later when we actually upgrade too.